### DISCUSSION TITLE
Not able to connect Exasol in Dbeaver.

### DISCUSSION THREAD

**Discussion by Clement0123456789** [dbeaver/dbeaver#18525] [Q&A]
The error message is: 
Error downloading driver libraries
Error resolving dependencies
  Maven artifact 'maven:/com.exasol:exasol-jdbc:RELEASE' not found

**Comment by LonwoLonwo** [comment]
Hello @Clement0123456789 

Thanks for the report.

As a workaround, for now, you can download the driver https://repo1.maven.org/maven2/com/exasol/exasol-jdbc/7.1.16/exasol-jdbc-7.1.16.jar and add it in the `Connection Setting -> Driver Settings -> Libraries` manually

**Comment by uslss** [comment]
unfortunately, cannot reproduce the issue.
@Clement0123456789 which dbeaver version do you use?

**Reply by Clement0123456789** [reply]
I use Dbeaver 22.3.
Thanks for your help.
I was able to solve the problem addind the driver manually and then by checking Load optional dependencies.

**Comment by melissagile** [comment]
Awesome, thanks so much for the help! So this looks like it has worked (followed your instructions as well as those listed [here](https://github.com/dbeaver/dbeaver/issues/18556) in a similar issue). The error I am getting now is as follows:

`java.io.IOException: TLS connection to host (10.150.210.11) failed: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target. If you trust the server, you can include the fingerprint in the connection string: 10.150.210.11/CA988988C015F859CB954AB9904FA62A5E466891352CD204843BB43A8BCE636F:8563. `

I assume I just need to add this last part to some permissions somewhere, but not sure where. Do you have any pointers perhaps?

**Reply by LonwoLonwo** [reply]
Hello @melissagile 
Maybe you can find something helpful here https://community.microstrategy.com/s/article/KB485556-How-to-connect-to-Exasol-using-TLS-certificate-through-MicroStrategy-m2021-Update-6?language=en_US

Also, we added a special checkbox in the Connection settings for this situation: https://github.com/dbeaver/dbeaver/pull/18593

**Comment by melissagile** [comment]
Hi Anastasiya,
Thank you for the support! Just wanted to follow up since we've found a
solution - it turned out to be an internal permissions issue with my
company. It works now :) Appreciate the help and have a lovely day!
